### ISSUE TITLE
Transparency in Zed

### ISSUE THREAD

**Issue by jmurphy4** [scottmckendry/cyberdream.nvim#115]
### Question or Suggestion

Hi there, love the theme. Have been trying to get it to work with Zed. The colors look right, but its not actually transparent? Any ideas on this? 

Thanks!

**Comment by scottmckendry** [comment]
Hello @jmurphy4!

Glad to hear you like the theme 🙂 I’m not super familiar with Zed and the extra was added by another contributor.

It looks like this is possible via overrides similar to the ones described in this comment:

https://github.com/zed-industries/zed/issues/5040#issuecomment-2053656538

I would happily merge a PR that implements this in a way the complements the cyberdream palette.

**Comment by jmurphy4** [comment]
Ahhhh gotcha. That makes sense. Let me take a stab at it

**Comment by scottmckendry** [comment]
Great stuff! Make sure you check out the contributing guide for tips on editing extras 🙂 

If you have any questions, let me know!

**Comment by jmurphy4** [comment]
```json
"experimental.theme_overrides": {
		"background.appearance": "blurred",
		"background": "#4c464290",
		"editor.background": "#00000030",
		"editor.gutter.background": "#00000030",
		"tab_bar.background": "#00000030",
		"terminal.background": "#00000030",
		"terminal.ansi.dim_black": "#FFFFFF",
		"editor.active_line_number": "#FFFFFF",
		"toolbar.background": "#00000030",
		"tab.active_background": "#28282820",
		"tab.inactive_background": "#00000000",
		"status_bar.background": "##00000030",
		"title_bar.background": "#00000030",
		"panel.background": "#00000030",
		"border.variant": "#00000000",
		"scrollbar.track.border": "#00000000",
		"editor.active_line.background": "#1e2124"
	},
```

I got it most of the way there, the only thing im missing is the terminal autocomplete, and I have no idea how to fix it lol

<img width="148" alt="PNG image" src="https://github.com/user-attachments/assets/4a57ceb6-1d1e-460d-b696-7cb37dd7e0d1">
<img width="1198" alt="Screenshot 2024-08-08 at 8 31 20 PM" src="https://github.com/user-attachments/assets/23e11f07-20a9-45eb-8a79-cc4ee69b7cdc">

**Comment by scottmckendry** [comment]
looking pretty slick! @byt3m4st3r, any ideas on how to get transparency working on the terminal?

**Comment by byt3m4st3r** [comment]
Hi @scottmckendry. The current implementation is experimental and as of now seems not stable. If you play around, you get a transparent terminal background by setting the terminal colour to a wrong colour value f.e. `terminal.background": "#00000000",` or `terminal.background": "#0000",` :-)

**Comment by byt3m4st3r** [comment]
> ```json
> "experimental.theme_overrides": {
> 		"background.appearance": "blurred",
> 		"background": "#4c464290",
> 		"editor.background": "#00000030",
> 		"editor.gutter.background": "#00000030",
> 		"tab_bar.background": "#00000030",
> 		"terminal.background": "#00000030",
> 		"terminal.ansi.dim_black": "#FFFFFF",
> 		"editor.active_line_number": "#FFFFFF",
> 		"toolbar.background": "#00000030",
> 		"tab.active_background": "#28282820",
> 		"tab.inactive_background": "#00000000",
> 		"status_bar.background": "##00000030",
> 		"title_bar.background": "#00000030",
> 		"panel.background": "#00000030",
> 		"border.variant": "#00000000",
> 		"scrollbar.track.border": "#00000000",
> 		"editor.active_line.background": "#1e2124"
> 	},
> ```

As I see, you use ZSH. If you use [zsh-autosuggestions](https://github.com/zsh-users/zsh-autosuggestions) you can set your completion colour to cyberdream grey with an env var like this: `ZSH_AUTOSUGGEST_HIGHLIGHT_STYLE=fg=#7b8496`.

**Comment by scottmckendry** [comment]
Appreciate you stepping in with the assist here @byt3m4st3r  🙂 Looks super clean!

As you've mentioned this is an experimental feature so I think we'll leave the change out of the main branch for now (sorry if you had plans for a PR @jmurphy4!)

We can revisit in the future when Zed theming is a bit more mature.

**Comment by jmurphy4** [comment]
Thanks for the replies guys!
Roger on the PR. No worries 🫡

**Comment by duychip0909** [comment]
Does anyone have problem on ubuntu "blurred" background.appearance?, it still transparency, any solution for this? @byt3m4st3r

**Comment by byt3m4st3r** [comment]
I recognized that I messed up the alpha values in the hex color codes from this comment https://github.com/scottmckendry/cyberdream.nvim/issues/115#issuecomment-2278437484

The hex color code has to follow the pattern #RRGGBBAA.

F.e. #0016181a has to be re-defined as #16181a00, but 00 would be fully transparent.

An alpha value for 20% transparency amounts to 33 and the corresponding color code is #16181a33.

Tested on Mac OS X and it works.

Maybe this solves your issue.
